sdcm/cluster.py

```diff
@@ -2786,6 +2791,21 @@ def _node_setup(self, node, seed_address):
         scylla_yaml_contents = scylla_yaml_contents.replace("cluster_name: 'Test Cluster'",
                                                             "cluster_name: '{0}'".format(self.name))
 
+        if node.is_addition:
```

Comment:
We should disable bootstrap when we create nodes to establish the cluster, if we want to add more nodes when the cluster already exists, then we should enable bootstrap.

So `addition` means not the first set of node.